fe/src/pages/index.vue

```diff
@@ -29,7 +27,55 @@
+  description:
+    "Lorem ipsum dolor sit amet consectetur. sectetur. sectetur. Amet senectus augue in urna.Lorem ipsum dolor sit amet",
+  ethValue: "0000.2",
+  daysLeft: 2410
+});
```

Comment:
This dummy object is quite complex and the image path is VERY long.
Let's create a dummy json file for this instead and destructure the object to be used in the template.
```suggestion
import CardSample from '~/public/card-sample.json'

 const {title, imgSrc, description, ethValue, daysLeft} = ref<CampaignCardProps>(CardSample)
```